UnityClient/Assets/Scripts/CameraSwitching.cs

```diff
@@ -7,17 +7,80 @@ public class CameraSwitching : MonoBehaviour
     private Camera _mainCamera;
```

Comment:
These hard-coded camera setups are not extensible. If the map ever changes, this design will immediately break and/or become hideous. Please consider redesigning with configurability and extensibility in mind.